contrib/intel_env.sh

```diff
@@ -33,3 +33,6 @@ export NSLOTS=16
 export PYTHON_EGG_CACHE=.eggs_cache
 export BIOBLEND_REPO_DIR=/cluster_share/Galaxy/api/bioblend
 export FASTQC_PERL_SCRIPT=/opt/Genomics/ohsu/dnapipeline/FastQC/fastqc
+export BCFTOOLS=/home/karthikg/broad/non_variant_db/bcftools/bcftools
```

Comment:
Will this tools be ported over to the new Austin/Portland clusters with the same path?